### ISSUE TITLE
.standalone_migrations file does not change the path for schema in structure.

### ISSUE THREAD

**Issue by DanEscamilla** [thuss/standalone-migrations#150]
This is my .standalone_migrations file:

~~~~
db:
    seeds: Memory/db/seeds.rb
    migrate: Memory/db/migrate
    schema: Memory/db/schema.rb
config:
    database: Memory/db/config.yml
~~~~

My version is standalone_migrations (5.2.6)

I get the following error when running `rake db:migrate --trace`.

~~~~
** Invoke db:migrate (first_time)
** Invoke db:load_config (first_time)
** Invoke environment (first_time)
** Invoke standalone:connection (first_time)
** Execute standalone:connection
** Execute environment
** Execute db:load_config
** Execute db:migrate
** Invoke db:_dump (first_time)
** Execute db:_dump
** Invoke db:schema:dump (first_time)
** Invoke db:load_config 
** Execute db:schema:dump
rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - /home/daniel/Documents/Ruby/CLI/db/schema.rb
/home/daniel/.rvm/gems/ruby-2.5.1/gems/activerecord-5.2.1/lib/active_record/railties/databases.rake:250:in `initialize'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/activerecord-5.2.1/lib/active_record/railties/databases.rake:250:in `open'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/activerecord-5.2.1/lib/active_record/railties/databases.rake:250:in `block (3 levels) in <top (required)>'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `block in execute'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `each'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `execute'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:213:in `block in invoke_with_call_chain'
/home/daniel/.rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/monitor.rb:226:in `mon_synchronize'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:193:in `invoke_with_call_chain'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:182:in `invoke'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/activerecord-5.2.1/lib/active_record/railties/databases.rake:68:in `block (2 levels) in <top (required)>'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `block in execute'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `each'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `execute'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:213:in `block in invoke_with_call_chain'
/home/daniel/.rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/monitor.rb:226:in `mon_synchronize'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:193:in `invoke_with_call_chain'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:182:in `invoke'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/activerecord-5.2.1/lib/active_record/railties/databases.rake:61:in `block (2 levels) in <top (required)>'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `block in execute'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `each'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:271:in `execute'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:213:in `block in invoke_with_call_chain'
/home/daniel/.rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/monitor.rb:226:in `mon_synchronize'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:193:in `invoke_with_call_chain'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/task.rb:182:in `invoke'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:160:in `invoke_task'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:116:in `block (2 levels) in top_level'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:116:in `each'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:116:in `block in top_level'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:125:in `run_with_threads'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:110:in `top_level'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:83:in `block in run'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:186:in `standard_exception_handling'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/lib/rake/application.rb:80:in `run'
/home/daniel/.rvm/gems/ruby-2.5.1/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
/home/daniel/.rvm/gems/ruby-2.5.1/bin/rake:23:in `load'
/home/daniel/.rvm/gems/ruby-2.5.1/bin/rake:23:in `<main>'
/home/daniel/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
/home/daniel/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => db:schema:dump
~~~~

In fact, when I create a directory "db" within my root directory it no longer throws an error and, it creates the schema.rb file there, while the config file is read from Memory/db/ and the database is properly created using migrations from Memory/db/migrate

**Comment by benbonnet** [comment]
same here; the specified schema path is ignored. had to create a db folder to get over this issue

**Comment by HassanTC** [comment]
hi @DanEscamilla, @benbonnet 
i solved the problem here https://github.com/thuss/standalone-migrations/pull/151
we need to merge this PR

**Comment by thuss** [comment]
I've published 5.2.7 which includes PR #151 

https://rubygems.org/gems/standalone_migrations/versions/
